modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -1366,10 +1379,10 @@ private void assertNotEqualsDataNodesForZone(int zoneId, @Nullable Set<String> c
         ));
     }
 
-    private void startDistributionZoneManager() throws NodeStoppingException {
-        distributionZoneManager.start();
+    private void startDistributionZoneManager() throws Exception {
+        deployWatchesAndUpdateMetaStorageRevision(metaStorageManager);
```

Comment:
And again, the order of deploying watches an starting dzm is probably incorrect.